src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs

```diff
@@ -26,13 +27,20 @@ void M()
 }";
             var (solution, locations) = CreateTestSolution(markup);
             var expected = CreateCommand(CSharpFeaturesResources.Use_implicit_type, locations["caret"].First());
+            var clientCapabilities = new LSP.ClientCapabilities()
```

Comment:
can we have a helper extension method to do this simpler? by the way, this doesnt work? jtoken["Experimental"]["supportsWorkspaceEdits"] = false?

you can get jtoken by JToken.FromObject(new Lsp.ClientCapabilities()) ?

Reply:
That doesn't seem any simpler, I'd first convert to a jtoken (JToken.FromObject), set the property, then convert it back to an actual client capability type to pass it in.  Seems easy enough to just create it like I did above.  

I can move it to a helper method, but this shouldn't be used anywhere but here.